### ISSUE TITLE
json api for rest api

### ISSUE THREAD

**Issue by jnchin314** [Chiv2-Community/chivstats.xyz-web#13]
Trying to get down the requirements for making a json api specifically for rest.  Normally you don't need things like words and texts and things like that.  just raw data

So I'm going to outline a few things and if i'm missing anything please let me know @gimmic 

1. Sideboard titles and urls
2. player search functionality
3. Player profile data
4. leaderboard data
5. (EDIT) "News" section data

Various things like wording, graphs, and format can be implemented directly in a completely seperate standalone js UI

**Comment by gimmic** [comment]
I'd also like to move the news content to a db table for easier updates outside of direct template editing.

**Comment by jnchin314** [comment]
> I'd also like to move the news content to a db table for easier updates outside of direct template editing.

We could also consider just moving the news stuff to the front end, since it would be easy to use as a yaml file or some form of data entry

Another idea is to leave it in the chiv-web backend and return it as json but also served from a text file with some form of data format.

**Comment by gimmic** [comment]
That's true. I had previously attempted to wrap Django management around
it.
We could use a user management system.

**Comment by jnchin314** [comment]
news section in yaml https://github.com/Chiv2-Community/chivstats.xyz-web/pull/16

either way (yaml or db) we would need to print the python news object
